### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by sdinot**
The aim of this pull request is to use [Github Actions](https://github.blog/2019-08-08-github-actions-now-supports-ci-cd/) as CI platform, [as proposed on Hipparchus forum](https://forum.orekit.org/t/new-continuous-integration-platform-proposal-for-hipparchus/756).

This new CI process brings two new functionalities:
* Publication of [QA Report](https://sonar.orekit.org/dashboard?id=org.hipparchus%3Ahipparchus-aggregator) on the Orekit instance of SonarQube
* Publication of [binary packages](https://packages.orekit.org/#browse/browse) on Orekit instance of Nexus.

### CODE REVIEW

File: hipparchus-parent/pom.xml [hunk 1]
```diff
@@ -491,6 +491,8 @@
     <hipparchus.hamcrest-library.version>1.3</hipparchus.hamcrest-library.version>
     <hipparchus.reflow-velocity-tools.version>1.1.1</hipparchus.reflow-velocity-tools.version>
     <hipparchus.velocity.version>1.7</hipparchus.velocity.version>
+    <!-- sonar related properties -->
+    <sonar.host.url>https://sonar.orekit.org/</sonar.host.url>
```
Comment (maisonobe) [inline hipparchus-parent/pom.xml:495]: I would just change the sonar.host.url property into hipparchus.sonar.host.url for the sake of consistency
with existing properties.

Comment (sdinot) [reply hipparchus-parent/pom.xml:495]: If I am not mistaken, the name of the property is imposed by the Sonar plugin for Maven.

Comment (sdinot) [reply hipparchus-parent/pom.xml:495]: I you prefer, I can pass the `sonar.host.url` property on the command line, through an environment variable:

```bash
mvn -B -f pom.xml sonar:sonar -Dsonar.login=$SONARQUBE_TOKEN -Dsonar.host.url=$SONARQUBE_HOST -Dsonar.branch.name=${GITHUB_REF##*/}
```

Comment (maisonobe) [reply hipparchus-parent/pom.xml:495]: If the plugin enforces the name, then we will use this name.

**Pull request merged**